CMakeLists.txt

```diff
@@ -170,7 +170,9 @@ if(CLR_CMAKE_PLATFORM_UNIX AND NOT DEFINED CLR_CROSS_COMPONENTS_BUILD)
 endif()
 
 if(CLR_CMAKE_PLATFORM_UNIX)
-    add_subdirectory(src/ToolBox/SOS/lldbplugin)
+    if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
```

Comment:
I assume this is just temporary until you make building the lldb plugin with GCC work, right?

Reply:
I thought lldb is clang specific. Let me undo this and move it to my hack.patch instead for temporary exclusion.

Reply:
This builds a plugin for lldb that allows you to use SOS commands to dump managed heap, objects, stack etc. There is nothing clang specific.